pkg/test/testmain.go

```diff
@@ -0,0 +1,27 @@
+package tester
+
+// A A is a string type
```

Comment:
Haha, this was originally `A is` but golint was complaining about it because "A" is technically an article. "An A is" satisfies golint, though. :-)